File: Section Reports/Layout/SubReport/C#/SubReport/rptMasterSub.cs
namespace GrapeCity.ActiveReports.Samples.SubReport
{
	/// <summary>
	/// Description of the outline of the  rptMasterSub.
	/// 
	/// </summary>
	public partial class rptMasterSub : GrapeCity.ActiveReports.SectionReport
	{
		public rptMasterSub()
		{
			//
			// ActiveReport designer support is required
			//
			//
			InitializeComponent();
			Data.OleDBDataSource dS = new Data.OleDBDataSource();
			dS.ConnectionString = Properties.Resources.ConnectionString;
			dS.SQL = "Select * from [order details] inner join products on [order details].productid = products.productID where [order details].orderID = <%OrderID%>";
			DataSource = dS;
		}
	}
}

File: Section Reports/Layout/SubReport/VB.NET/SubReport/rptMasterSub.vb
Public Class rptMasterSub
	Inherits GrapeCity.ActiveReports.SectionReport
	Dim _ds As GrapeCity.ActiveReports.Data.OleDBDataSource
	Public Sub New()
		MyBase.New()
		InitializeComponent()
		Dim dS As New Data.OleDBDataSource()
		dS.ConnectionString = My.Resources.ConnectionString
		dS.SQL = "Select * from [order details] inner join products on [order details].productid = products.productID where [order details].orderID = <%OrderID%>"
		DataSource = dS
	End Sub
End Class
